pyproject.toml

```diff
@@ -15,28 +15,30 @@ dynamic = ["version"]
 dependencies = [
   'click>=8.0',
   'countryinfo>=0.1.2',
-  'lb_matching_tools==v2024.01.30.1',
+  'lb_matching_tools>=v2024.01.30.1',
   'liblistenbrainz>=v0.5.5',
   'more_itertools',
   'mutagen==1.46.0',
-  'nmslib==2.1.1',
-  'peewee==3.17.0',
+  'peewee>=3.17.0',
   'psycopg2-binary>=2.9.3', 
-  'py-sonic==1.0.0',
+  'py-sonic>=1.0.0',
   'python-dateutil>=2.8.2',
-  'regex==2023.6.3',
+  'regex>=2023.6.3',
   'requests>=2.31.0',
-  'scikit-learn==1.2.1',
+  'scikit-learn>=1.2.1',
```

Comment:
what's scikit-learn used for?

Reply:
The data that gets passed to the fuzzy matcher gets prepare by a call to scikit.learn. (see fuzzy_index.py)